mod_reforged/hooks/skills/actives/throw_smoke_bomb_skill.nut

```diff
@@ -0,0 +1,31 @@
+::mods_hookExactClass("skills/actives/throw_smoke_bomb_skill", function(o) {
+	local getTooltip = o.getTooltip;
+	o.getTooltip = function()
+	{
+		local ret = getTooltip();
+        foreach (index, entry in ret)
+        {
+            if (entry.id != 5) continue;
+			if (entry.text.find("Increases Ranged Defense by ") == null) continue;	// Vanilla has two entries with id 5. That's why we check for a phrase aswell
```

Comment:
```squirrel
if (entry.id != 5 || entry.text.find("Ranged Defense") == null)
    continue;
```

In fact for this such a small case the readability is way better if you do:
```squirrel
if (entry.id == 5 && entry.text.find("Ranged Defense") != null)
{
    ret.remove(index);
    break;
}
```
Either way, please combine both conditions into one `if` block.

Reply:
I wouldn't say `way better`
But I'm ok changing it in this case